javascript/express/security/audit/remote-property-injection.yaml

```diff
@@ -0,0 +1,34 @@
+rules:
+- id: remote-property-injection
+  message: |
+    Bracket object notation with user input is present, this might allow an attacker to access all properties of the object and even it's prototype. Use literal values for object properties.
+  metadata:
+    owasp: 'A1: Injection'
+    cwe: 'CWE-400: Uncontrolled Resource Consumption'
```

Comment:
Is this CWE correct?

Reply:
good question

Reply:
How about CWE-1321: Improperly Controlled Modification of Object Prototype Attributes ('Prototype Pollution')?

Reply:
yeah lets change it to 1321

Reply:
Thank you :pray: